src/components/RegistrationTable/EditCell.tsx

```diff
@@ -11,13 +11,38 @@ import { Row, Table } from "@tanstack/react-table"
 import { Attendee } from "./columns"
 import Image from "next/image";
 import PopoutIcon from "../../../public/assets/icons/popout_icon.svg";
+import UserInfo from "./editCellPopUp.tsx/userInfo"
+import UserResponses from './editCellPopUp.tsx/userResponses'
+import { RegQuestionData, UserQuestionIDs, UserResponse } from '@/types'
+import router from 'next/router'
 
 interface EditCellProps {
     row: Row<Attendee>
     table: Table<Attendee>
 }
 
-export const EditCell: React.FC<EditCellProps> = ({ row }) => {
+export const EditCell: React.FC<EditCellProps> = ({ row, table }) => {
+    const [questions, setQuestions] = useState<RegQuestionData[]>([]);
+    const [responses, setResponses] = useState<{ [key: string]: UserResponse }>({});
+
+    useEffect(() => {
+        if (router.isReady) {
+            const eventId = router.query.eventId as string;
+            const attendeeId = row.original.id;
+            const eventYear = router.query.year as string;
+
+            if (eventId && attendeeId && eventYear) {
+                fetchQuestionsAndResponses(attendeeId, eventId, eventYear).then(data => {
+                    setQuestions(data.questions);
+                    setResponses(data.responses);
+                }).catch(error => {
+                    console.error("Error fetching questions and responses:", error);
+                });
+            }
+        }
+    }, [row.original.id]);
```

Comment:
Is this data not already available in `row`? Not sure if it's necessary to fetch again